src/components/dropdowns/primary-action-dropdown/primary-action-dropdown.js

```diff
@@ -230,7 +230,7 @@ const PrimaryActionDropdown = props => {
     return () => {
       window.removeEventListener('click', handleGlobalClick);
     };
-  }, [handleGlobalClick]);
+  });
```

Comment:
We want it to run once not on each render right. So empty array?

Reply:
good call